bindings/rust/evmc-sys/Cargo.toml

```diff
@@ -13,4 +13,4 @@ categories = ["external-ffi-bindings"]
 edition = "2018"
 
 [build-dependencies]
-bindgen = "0.54.0"
+bindgen = "0.59"
```

Comment:
This bumps the minimum Rust requirement from 1.37 to 1.47, which is fine but the README must be updated accordingly.

Reply:
Moved this change to #638.